### ISSUE TITLE
Suggestion: Archer/Bowman and his partners

### ISSUE THREAD

**Issue by kihgy** [TangoTek/TekTopia-Community#1025]
Archer could be a separated profession or just a career for the guards, I also have a few ideas of his providers
Fisherman: the villager provides string and damaged bows on occasions.
Dog whisperer: Command a army of tamed dogs, and send them after skeletons, collect bones and arrows, heal and breed the dogs using rotten fleshes from zombies, his pack have a maximum of 10 dogs

**Comment by kirknay** [comment]
I agree that these professions would fit perfectly in the mod. I wonder if Tango would look into them when he comes back to the mod?
